### PR TITLE
R2018b blog post

### PR DESCRIPTION
**Pull request by tn12787**
A blog post for the R2018b release.

It also reverts some unwanted changes that came from the updates to the doc CSS in january.

### CODE REVIEW

**Comment by tn12787** [comment]
preview: https://www.cyberbotics.com/doc/blog/Webots-2018-b-release?version=feature-2018b-blog-post

**Comment by tn12787** [comment]
👍

File: blog/Webots-2018-b-release.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+As a result of implementing only what we need, and focusing on using modern techniques introduced with OpenGL 3+, Webots with WREN is much faster than with OGRE.
+OGRE 1.9 (the version we used) is a CPU-bound renderer, which is bad news for Webots, which is optimised for single-thread performance.
+Ideally, a 3D renderer for use with Webots should be GPU bound, and be as CPU-overhead minimal as possible, to provide "room" for the simulation's CPU-sided simulation workload (robot controllers, physics, GUI) as possible.
+WREN is extremely CPU-light, and is GPU-bound for rendering.
+This has allowed us to improve world performance considerably.
+
+
+To illustrate this in practical terms, the simulation speedometer in "Run" mode is used as our performance indicator, to show the real-life implications of faster rendering in a Webots simulation.
+The following benchmarks were performed on Windows 10, 64-bit with an Intel 6700k processor @ 3.40GHz and an NVIDIA GTX 970 graphics cards on some familiar webots worlds.
```
Comment (DavidMansolino) [inline blog/Webots-2018-b-release.md]: Shouldn't Webots be capitalized?

File: blog/Webots-2018-b-release.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+As always, for a comprehensive list of changes please refer to the ChangeLog, found [here](https://www.cyberbotics.com/dvd/common/doc/webots/ChangeLog.html).
+
+## WREN: Webots Renderer
+
+For the past decade, Webots has used the [Object-Oriented Graphics Rendering Engine](https://www.ogre3d.org/) (OGRE) to handle 3D rendering. 
+We encountered many roadblocks with OGRE, especially in modernizing our worlds and achieving a greater level of realism, and as such decided that it was better for us to implement our own bespoke rendering engine - from scratch!
+This allows us to implement and use only the features we need / desire, giving us a very lightweight and efficient renderer.
+
+
+We've spent the past year doing just this, and are proud to present our own bespoke OpenGL 3.3 based renderering engine, WREN (Webots RENderer).
```
Comment (DavidMansolino) [inline blog/Webots-2018-b-release.md]: just this  => it a bit exaggerated and might let think that this is the only new feature of this release.

File: blog/Webots-2018-b-release.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+The following benchmarks were performed on Windows 10, 64-bit with an Intel 6700k processor @ 3.40GHz and an NVIDIA GTX 970 graphics cards on some familiar webots worlds.
+
+|    World    | R2018a | R2018b | speedup |
+|:-----------:|:------:|:------:|:-------:|
+|   city.wbt  |  2.82x |  7.60x |  2.70x  |
+| battery.wbt |  35.1x |  43.6x |  1.24x  |
+| create.wbt  | 30.4x  |  52.4x |  1.72x  |
+
+
+As you can see from these tables, even with robot controller taking a large part of the timestep in these worlds, all of them run considerably faster than before, with a greater speedup in complex worlds due to higher rendering demand.
```
Comment (DavidMansolino) [inline blog/Webots-2018-b-release.md]: these tables => there is only one table no?

File: blog/Webots-2018-b-release.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+### Physically-Based Rendering
+
+%figure "A model of a telephone, from the GLTF sample library, rendered in Webots."
+![telephone](images/telephone.png)
+%end
+
+In a bid to make Webots look as realistic as possible, we have implemented a Physically-Based Rendering pipeline into Webots.
+Physically based rendering relies on improved lighting models, and the use of several additional non-colour textures (normal maps, roughness maps and metalness maps) to represent a material's interactions with light to a very high degree of realism.
+This is useable with the new PBRAppearance and Cubemap nodes.
```
Comment (DavidMansolino) [inline blog/Webots-2018-b-release.md]: As a general comment, we should link to the corresponding doc (e.g. here to `PBRAppearance` and `Cubemap` nodes in reference manual).

File: blog/Webots-2018-b-release.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+%end
+
+This feature makes resetting even the most complex simulations instantaneous.
+
+---
+
+## New PROTOs
+
+For more realistic and immersive vehicle simulations, we have expanded our available building models and street furniture extensively, with more than 80 new models.
+Below is a preview of just some these new assets.
```
Comment (DavidMansolino) [inline blog/Webots-2018-b-release.md]: just some these new assets. => some **of** these

File: blog/Webots-2018-b-release.md (hunk 2 above)
Comment (tn12787) [reply blog/Webots-2018-b-release.md]: This has a different meaning to what you might think in english, it is used to clarify that the previously-described action was indeed undertaken.

But I see the confusion, I'll change it so it's easier to understand.

**Comment by omichel** [comment]
:eyes:

File: blog/Webots-2018-b-release.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
+# Version R2018b Released
+
+<p id="publish-data">By Tom Norton - xxth June 2018</p>
+
+---
+
+It's that time of year again!
+Today marks the release of the all-new Webots R2018b, packed with some game-changing features.
+As always, for a comprehensive list of changes please refer to the ChangeLog, found [here](https://www.cyberbotics.com/dvd/common/doc/webots/ChangeLog.html).
```
Comment (omichel) [inline blog/Webots-2018-b-release.md]: We should link to https://cyberbotics.com/change_log instead (looks better).

File: blog/Webots-2018-b-release.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+---
+
+## Extra Goodies
+
+Webots R2018b supports Ubuntu 18.04 LTS out of the box.
+
+The [Speaker API](../reference/speaker.md#speaker-functions) can now load and play all major audio formats.
+
+If you don't want to run a controller for your Robot node, you can now simply use "" as the controller name, to avoid running the void controller unnecessarily.
```
Comment (omichel) [inline blog/Webots-2018-b-release.md]: I would say "simply use an empty string ("") as the controller name" to be more explicit.

File: blog/Webots-2018-b-release.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+
+Projection mode, rendering mode, optional renderings and disable selection are now saved per world instead of globally.
+
+**Go and download Webots R2018b, so you don't miss out on all these great new features!**
```
Comment (omichel) [inline blog/Webots-2018-b-release.md]: I would add a link to https://cyberbotics.com/#download here.

**Comment by tn12787** [comment]
👍

**Comment by tn12787** [comment]
I have added next post / previous post buttons to the blog, previewable [here.](https://www.cyberbotics.com/doc/blog/Webots-2018-b-release?version=feature-2018b-blog-post)

**Comment by tn12787** [comment]
👍

**Comment by fabienrohrer** [comment]
That's really good.

**Pull request merged**